web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1108,6 +1108,8 @@ $.extend(fixmystreet.set_up, {
         hide('.js-new-report-user-hidden');
         show('.js-new-report-user-shown').then(function(){
             focusFirstVisibleInput();
+            // run now to protect staff fields when adding an update:
+            fixmystreet.staff_set_up.contribute_as();
```

Comment:
`staff_set_up` does not exist at this point if you are not logged in as staff, and will error (not a JS error because the deferred swallows it). This is not the correct place to do this (note no other use of staff in this file), but in staff.js in the `display:report` handler seems a good place to call it.